client/components/gsuite/gsuite-features/index.jsx

```diff
@@ -80,14 +80,12 @@ const GSuiteFeatures = ( { compact, domainName, productSlug, type } ) => {
 			/>
 			<GSuiteSingleFeature
 				title={
-					compact ? translate( 'Video and voice calls' ) : translate( 'Connect with your team' )
+					compact ? translate( 'Video calls' ) : translate( 'Connect with your team' )
 				}
 				description={
 					compact
 						? undefined
-						: translate(
-								'Use text chats, voice calls, or video calls, with built in screen sharing.'
-						  )
+						: translate( 'Use text chats or video calls, with built in screen sharing.' )
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Use text chats, voice calls, or video calls, with built in screen sharing.' )` *ES Score: 10*